uber_sante/controllers/booking_controller.py

```diff
@@ -29,25 +32,28 @@ def book():
     # TODO: connect the call to the scheduler class to try reserving the availability (line 51)
     # TODO: connect the call to the booking_service to create the booking in the Booking table (line 54)
 
-        appointment_id = request.args.get('appointment_id')
+        availability_id = request.args.get('availability_id')
         patient_id = request.args.get('patient_id')
 
-        if appointment_id is None:
-            return jsonify('No appointment specified'), 400
+        if availability_id is None:
+            return js.create_json(data=None, message="No appointment specified", return_code=js.ResponseReturnCode.CODE_400)
         if patient_id is None:
-            return jsonify('No patient specified'), 400
+            return js.create_json(data=None, message="No patient specified", return_code=js.ResponseReturnCode.CODE_400)
```

Comment:
Remove this _if_ because the conditions above (in my comments) will account for this